.yamato/disable-burst-if-requested.py

```diff
@@ -0,0 +1,93 @@
+import argparse
```

Comment:
nit: I'm assuming this was copied from the SDK repo as well? could you write a source for this in a quick comment at the top? just so we know who to poke about this